srml/council/src/voting.rs

```diff
@@ -168,11 +201,17 @@ impl<T: Trait> Module<T> {
 		<VetoedProposal<T>>::remove(proposal);
 	}
 
+	/// Perform and return the tally result of a specific proposal.
+	///
+	/// The returned tuple is: `(approves, rejects, abstainers)`.
 	fn take_tally(proposal_hash: &T::Hash) -> (u32, u32, u32) {
```

Comment:
as per previous comment, consider changing to `fn take_tally(proposal_hash: &T::Hash) -> (YaysCount, NaysCount, AbstainsCount) {`, and then remove the comment `The returned tuple is: (approves, rejects, abstainers).`, respectively. since the we now have self-describing code.

Reply:
What do you recommend `YaysCount` etc. to be actually as a type? a type alias for each of them? 

And, best to not change the code in a doc PR.